spec/services/db/add_document_verification_and_selfie_costs_spec.rb

```diff
@@ -30,22 +31,38 @@
     described_class.new(
       user_id: user_id,
       service_provider: service_provider,
+      liveness_checking_enabled: liveness_checking_enabled,
     )
   end
+  context 'when livness check is enabled' do
+    it 'has costing for front, back, selfie, and result when billed' do
```

Comment:
Just curious: how did you know to add pieces about cost in? I don't see it mentioned in the ticket, and I don't think I would have known to look for this.

Reply:
@night-jellyfish from existed code that was removed.